### PR TITLE
feat: differential gauge

### PR DESCRIPTION
**Pull request by leemind**
Use config 'differential' and min must == -max (e.g. -50 & +50)

### CODE REVIEW

File: justgage.js [hunk 1]
```diff
@@ -543,6 +549,39 @@
...
+
+        Xo = Cx + Ro * Math.cos(alpha);
+        Yo = Cy - Ro * Math.sin(alpha);
+        Xi = Cx + Ri * Math.cos(alpha);
+        Yi = Cy - Ri * Math.sin(alpha);
+
+        So = (value < 0) ? 1 : 0
+        Si = (value < 0) ? 0 : 1
+
+        //console.log(`Value: ${value} So: ${So} Si: ${Si}`)
```
Comment (robertsLando) [inline justgage.js:574]: Remove comments

File: justgage.js [hunk 2]
```diff
@@ -543,6 +549,39 @@
...
+
+        path = "M" + Cx + "," + (Cy - Ri) + " ";
+        path += "L" + Cx + "," + (Cy - Ro) + " ";
+        path += "A" + Ro + "," + Ro + " 0 0 " + Si + " " + Xo + "," + Yo + " ";
+        path += "L" + Xi + "," + Yi + " ";
+        path += "A" + Ri + "," + Ri + " 0 0 " + So + " " + Cx + "," + (Cy - Ri) + " ";
+        path += "Z ";
+
+        //console.log(path)
```
Comment (robertsLando) [inline justgage.js:583]: Same

File: justgage.js [hunk 3]
```diff
@@ -955,14 +995,16 @@
     if (obj.config.reverse) {
       rvl = obj.config.max * 1 + obj.config.min * 1 - obj.config.value * 1;
     }
+    console.log("animate?",rvl)
```
Comment (robertsLando) [inline justgage.js:998]: Same

**Comment by leemind** [comment]
> Also please add the example to: https://github.com/toorshia/justgage/tree/master/docs/examples
> 
> name it `differential.html`

Done

**Comment by robertsLando** [comment]
Fixes #345

**Review by robertsLando — CHANGES_REQUESTED** [review]
Please run `npm run lint-fix` command to fix lint issues

**Pull request merged**